skatepark/sass/block-patterns/_paragraph-with-quote.scss

```diff
@@ -0,0 +1,15 @@
+.paragraph-with-quote {
+    h4 {
+        margin: 0 0 calc( var(--wp--custom--margin--vertical) * 0.89 ) 0;
+    }
+
+    div:nth-child(2) {    
+        > p:first-of-type {
+            text-indent: 9.15em;
```

Comment:
I think you can use [this](https://github.com/Automattic/themes/blob/trunk/skatepark/sass/block-styles/_indented-paragraph.scss#L1) on the paragraph inside the block pattern instead of targeting the paragraph with CSS here.

Reply:
Oh nice, thanks!